server/routes/import_wizard.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Import Wizard routes"""
+
+from flask import Blueprint, render_template
+
+bp = Blueprint('import_wizard', __name__, url_prefix='/import')
+
+
+@bp.route('/')
+def main():
+    return render_template('/import_wizard.html')
```

Comment:
Should only allow this in autopush for now?

Reply:
yes, you're right, done